Formula/guile.rb

```diff
@@ -48,6 +49,15 @@ class Guile < Formula
     cause "Segfaults during compilation"
   end
 
+  # http://debbugs.gnu.org/cgi/bugreport.cgi?bug=23870
+  # https://github.com/Homebrew/homebrew-core/issues/1957#issuecomment-229347476
+  if MacOS.version >= :el_capitan
```

Comment:
> This comment makes me wonder if we should mandate CLT installation with Xcode 8 on El Cap, presuming that does actually resolve the issue.

It does, and I think that's a good idea. This is going to get ugly fast otherwise.

Given that this only shifts the problem around, let's avoid applying the patch on El Cap.